src/services/serverless.js

```diff
@@ -0,0 +1,47 @@
+const fs = require("fs");
+const YAML = require("js-yaml");
+
+class Serverless {
+  constructor() {
+    const ymlPath = "./serverless.yml";
```

Comment:
some users will have this installed globally, rather than a dev dependency, so checking if the `.option("-l, --location <location>", "location of your serverless project")` option is set will help.